### ISSUE TITLE
audio data buffer is always zero, why?

### ISSUE THREAD

**Issue by fcgll520** [danielpaulus/quicktime_video_hack#48]
the h264 file is fine, but the wav is always zero buffer, i can't find why?please give me some suggestion.
Thanks.

**Comment by danielpaulus** [comment]
Hey there 👋 
That is weird, I just tried my iPhone X and even when I don't play any sound on the device, still I get a wave file created. Can you run `go run main.go record out.h264 out.wav -v` and post the logs here? Also please let me know which device and version you are using :-)

**Comment by fcgll520** [comment]
thanks，i found the reason.
Another problem,
why iphone's showing time is changed(like, 09:41) when usb connecting?

**Comment by danielpaulus** [comment]
Yes and also it removes the provider info and changes the battery status that is displayed I think. That is Apple's default behavior, I guess they don't like mobile vendors and empty batteries in their presentation :-D As for the time, that is when the first iPhone was released or something.

**Comment by fcgll520** [comment]
When playing music in iphone's music app, it changes iphone->system capture mode, and then,  the receiving audio is just silence. Know why?
![image](https://user-images.githubusercontent.com/8245015/80555072-bd139900-8a01-11ea-8d17-faee06ccb3f9.png)
